utils/utils.go

```diff
@@ -29,3 +30,11 @@ func StringInSlice(a string, list []string) bool {
 	}
 	return false
 }
+
+func IsCommandAvailable(name string) bool {
+	cmd := exec.Command(name, "--help")
```

Comment:
NIT: I was wondering if exec.LookupPath("rpm") does the same job.

Reply:
I fixed it. Thanks.